utils/debug/Debug.js

```diff
@@ -28,6 +29,7 @@ function Debug(view, datDebugTool, chartDivContainer) {
     this.charts.push(new ThreeStatsChart('three-info', view.mainLoop.gfxEngine.renderer));
 
     const charts = this.charts;
+    const tileLayer = view.tileLayer || view.wgs84TileLayer || view.baseLayer;
```

Comment:
Note: this is why the PR https://github.com/iTowns/itowns/pull/546 introduced `view.baseLayer` for all views